.github/workflows/lint.yaml

```diff
@@ -2,9 +2,21 @@ name: lint
 
 on:
   pull_request:
-    branches: [main]
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
   push:
-    branches: [main]
+    branches:
+      - main
+      - '23.lts.1\+'
```

Comment:
Let's not lint release branches

Reply:
no bug id check for release branches either?

Reply:
I don't think it's necessary, no

Reply:
done